Changes

```diff
@@ -100,6 +100,10 @@ Working version
   (Greta Yorsh, review by Xavier Leroy, Guillaume Melquiond and
   Gabriel Scherer)
 
+- #11026, #11858: Rename the type of the accumulator of fold functions
+  to 'acc (Valentin Gatien-Baron, report by Francois Berenger, review
+  by Gabriel Scherer)
```

Comment:
Author + reviewers should go on a separate line.

Reply:
I went ahead and merged, and fixed this in trunk.